src/main/java/com/example/bank/account/Account.java

```diff
@@ -0,0 +1,49 @@
+package com.example.bank.account;
+
+import com.example.bank.registration.jpa.Client;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+@Entity
+/**  W columnNames podajemy nazwę pola, czy zmapowanej kolumny w bazie danych? */
+@Table(uniqueConstraints = @UniqueConstraint(columnNames = "account_number"))
+@Getter
+@Setter
+@NoArgsConstructor
+public class Account {
+    @Id
+    @SequenceGenerator(name = "account_sequence", sequenceName = "account_sequence", allocationSize = 10)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "account_sequence")
+    private Long id;
+    /**  Czy id może być typu String i co z inkrementacją tego pola?? Jaki to będzie GenerationType?  */
+    @Column(name = "account_number", nullable = false)
+    @NotBlank(message = "Account number is mandatory")
+    //TODO pattern
+    private String accountNumber;
```

Comment:
pomysl nad tym zeby przy tworzeniu konta zrobic jakis generator numeru kolejnego lub losowego ale spelniajacego warunki bycia numerem konta. Poczytaj o tym bo na pewno jest pierdyliard zasad co numer musi miec. Jakas czesc to bank konkretny (to sobie wymysl swoj ten fragment) a inne tez costam znacza